lib/resources/dstu2/document_reference.yaml

```diff
@@ -35,7 +35,9 @@ fields:
         }
       ]
     }
-  note: The type must include a loinc coding.
+  note: The type must include a LOINC or a proprietary coding. If multiple codings are provided they must be either all LOINC codings or all proprietary codings.
```

Comment:
> If multiple codings are provided

multiple types are not supported. Cardinality is 1..1

Reply:
the **type** CodeableConcept is 1..1 but CodeableConcept.coding is 0..*

Reply:
looking further, we need to communicate that when multiple proprietary codings are passed in, the first will be selected

Reply:
After an offline discussion with @ericsornson, it has been decided that rather than communicating about logical decisions that take place in the code when multiple proprietary codings are passed in,  it would be better to convey only one proprietary coding can be passed.